app/routes.py

```diff
@@ -0,0 +1,66 @@
+import os.path
+# using werkzeug.exceptions in order to get rid of pyramid
+# from pyramid.httpexceptions import HTTPBadRequest and other flask stuff
+# from pyramid.view import view_config
+# from pyramid.response import Response
+from flask import Response
+import six  # for compatibility Python 2 / 3
+from PIL import Image
+from app import app
+from app.helpers import check_color_channels
+from app.helpers import check_path
+
+# Python2/3 compatibility
+if six.PY3:
+    from io import BytesIO
+else:
+    from StringIO import StringIO
+
+
+@app.route('/checker')
+@app.route('/checker/')
+def checker_page():
+    """
+    Just a route for the health check
+    :return: OK with a 200 status code
+    """
+    return "OK", 200
+
+
+@app.route('/color/<r>,<g>,<b>/<filename>/', methods=['GET'])
+@app.route('/color/<r>,<g>,<b>/<filename>', methods=['GET'])
```

Comment:
Thanks for that hint!
I implemented that. Now it looks like:
`@app.route('/<int:r>,<int:g>,<int:b>/<string:filename>', methods=['GET'])`

Only problem yet to be resolved is, how to raise a specific error message in case non-integers are passed.
With the check-function used before, it was possible.
Now a generic 404 will be raised.